Manual (web-interface) and automatic (Gradle, API) deploy have different minimum character limit for Changelog/Release Bulletin.

**Describe the bug**
Manual (web-interface) and automatic (Gradle, API) deploy have different minimum character limit for Changelog/Release Bulletin. Web-interface allows to have any minimum length, but if you try to automatically deploy through gradle, it will error out with "Content too short" message.

**To Reproduce**
Steps to reproduce the behavior:
1. Try to manually add version to project. In "New project release" page, edit "Release Bulletin" section. Type any text with 1-14 character length (For example, "1"). Site will allow it.
2. Open or create OreTestPlugin project
3. Add or change `changelog='1'` (or any string with length of 1-14 characters) to oreDeploy section.
4. Execute `oreDeploy` task.
5. See error:
```
:oreDeploy
Publishing oretest to https://ore.spongepowered.org.
  Recommended: false
  Channel: snapshot
[failure] 400 Bad Request
* changelog
  - Content too short.
:oreDeploy FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':oreDeploy'.
> Deployment failed.
```

**Expected behavior**
A consistent minimum character limits in both web-interface and API.

**Environment**
Any

**Additional context**
n/a

--- a/apiV2/app/controllers/apiv2/ApiV2Controller.scala
+++ b/apiV2/app/controllers/apiv2/ApiV2Controller.scala
@@ -487,7 +487,6 @@ class ApiV2Controller @Inject()(factory: ProjectFactory, val errorHandler: HttpE
           .orElse(pluginInfoFromFileF)
           .toRight("No or invalid plugin info specified")
           .subflatMap(s => parser.decode[DeployVersionInfo](s).leftMap(_.show))
-          .ensure("Description too short")(_.description.forall(_.length < Page.minLength))
           .ensure("Description too long")(_.description.forall(_.length > Page.maxLength))
           .leftMap(e => BadRequest(ApiError(e)))
 

